.storybook/preview.js

```diff
@@ -24,11 +19,13 @@ export const parameters = {
 				? 0
 				: a[1].id.localeCompare(b[1].id, undefined, { numeric: true })
 	}
+	// viewMode: 'docs'
```

Comment:
No commented-out code please. Either remove the whole code block or remove the comment itself. 
```suggestion
```
```suggestion
	viewMode: 'docs'
```